src/Aspire.ProjectTemplates/templates/aspire-py-starter/.template.config/template.json

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Aspire",
+    "Cloud",
+    "Web",
+    "Web API",
+    "API",
+    "Service",
+    "Python",
+    "JavaScript"
+  ],
+  "name": "Aspire Python Starter App",
+  "defaultName": "AspirePyApp",
+  "description": "A project template for creating an Aspire app with a Python backend API service and a JavaScript frontend using Vite and React.",
+  "shortName": "aspire-py-starter",
+  "sourceName": "AspirePyApplication.1",
+  "preferNameDirectory": true,
+  "tags": {
+    "language": "C#",
+    "type": "solution",
+    "editorTreatAs": "solution",
+    "aspire-13.0-tfms": "net8.0;net9.0;net10.0"
+  },
+  "precedence": "9000",
+  "identity": "Aspire.PyStarter.CSharp.8.0",
+  "thirdPartyNotices": "https://aka.ms/dotnet/aspire/third-party-notices",
+  "groupIdentity": "Aspire.PyStarter",
+  "guids": [
+    "A1B2C3D4-E5F6-7890-ABCD-EF1234567890",
+    "B2C3D4E5-F678-90AB-CDEF-123456789012"
+  ],
+  "sources": [
+    {
+      "source": "./13.0",
+      "target": "./"
+    }
+  ],
+  "symbols": {
+    "Framework": {
+      "type": "parameter",
+      "description": "The target framework for the project.",
+      "datatype": "choice",
+      "choices": [
+        {
+          "choice": "net8.0",
+          "description": "Target net8.0"
+        },
+        {
+          "choice": "net9.0",
+          "description": "Target net9.0"
+        },
+        {
+          "choice": "net10.0",
+          "description": "Target net10.0"
+        }
+      ],
+      "replaces": "net8.0",
+      "defaultValue": "net9.0"
```

Comment:
I think we can remove this entire option as this uses a file-based AppHost so only works on .NET 10